commands/bot_e2e_test.go

```diff
@@ -198,7 +198,7 @@ func (s *MmctlE2ETestSuite) TestBotEnableCmd() {
 		s.Require().Nil(appErr)
 
 		err := botEnableCmdF(s.th.Client, &cobra.Command{}, []string{newBot.UserId})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```

Comment:
Please check this with `s.Require().Error(err)`, it makes the test messages much easier to read.